shared/team-building/container.js

```diff
@@ -0,0 +1,198 @@
+// @flow
+import logger from '../logger'
+import React from 'react'
+import * as I from 'immutable'
+import TeamBuilding from '.'
+import * as TeamBuildingGen from '../actions/team-building-gen'
+import {type TypedState, compose, connect, setDisplayName, createSelector} from '../util/container'
+import {PopupDialogHoc} from '../common-adapters'
+import {parseUserId} from '../util/platforms'
+import {followStateHelperWithId} from '../constants/team-building'
+import type {ServiceIdWithContact, User} from '../constants/types/team-building'
+
+// TODO
+// * there's a lot of render thrashing going on. using keyboard arrows is kinda slow becuase of it.
+// * Limit the highlight index to the max lenght of the list
+
+type OwnProps = {
```

Comment:
i think we should discuss which parts the owning store control and which parts the team building component control. i think we want the component to have more of its own state and handle more of this but split out the sourcing of the data and whatnot to chat (and others)

Reply:
Currently the only things the store gives is `searchResults`, `teamSoFar` and `searchResultsCount`